python/hail/methods/qc.py

```diff
@@ -29,58 +30,43 @@ def sample_qc(dataset, name='sample_qc') -> MatrixTable:
-    +--------------------------+---------+------------------------------------------------------+
-    | ``n_singleton``          | int64   | Number of private alleles                            |
-    +--------------------------+---------+------------------------------------------------------+
-    | ``n_transition``         | int64   | Number of transition (A-G, C-T) alternate alleles    |
-    +--------------------------+---------+------------------------------------------------------+
-    | ``n_transversion``       | int64   | Number of transversion alternate alleles             |
-    +--------------------------+---------+------------------------------------------------------+
-    | ``n_star``               | int64   | Number of star (upstream deletion) alleles           |
-    +--------------------------+---------+------------------------------------------------------+
-    | ``n_non_ref``            | int64   | Sum of ``n_het`` and ``n_hom_var``                   |
-    +--------------------------+---------+------------------------------------------------------+
-    | ``r_ti_tv``              | float64 | Transition/Transversion ratio                        |
-    +--------------------------+---------+------------------------------------------------------+
-    | ``r_het_hom_var``        | float64 | Het/HomVar call ratio                                |
-    +--------------------------+---------+------------------------------------------------------+
-    | ``r_insertion_deletion`` | float64 | Insertion/Deletion allele ratio                      |
-    +--------------------------+---------+------------------------------------------------------+
-    | ``dp_mean``              | float64 | Depth mean across all calls                          |
-    +--------------------------+---------+------------------------------------------------------+
-    | ``dp_stdev``             | float64 | Depth standard deviation across all calls            |
-    +--------------------------+---------+------------------------------------------------------+
-    | ``gq_mean``              | float64 | The average genotype quality across all calls        |
-    +--------------------------+---------+------------------------------------------------------+
-    | ``gq_stdev``             | float64 | Genotype quality standard deviation across all calls |
-    +--------------------------+---------+------------------------------------------------------+
-
-    Missing values ``NA`` may result from division by zero. The empirical
-    standard deviation is computed with zero degrees of freedom.
+    If `mt` contains an entry field `DP` of type :py:data:`.tint32`, then the
+    field `dp_stats` is computed. If `mt` contains an entry field `GQ` of type
+    :py:data:`.tint32`, then the field `gq_stats` is computed. Both `dp_stats`
+    and `gq_stats` are structs with with four fields:
+
+    - `mean` (``float64``) -- Mean value.
```

Comment:
The example above needs to change `dataset.sample_qc.dp_mean` to `dataset.sample_qc.dp_stats.mean`.